frontend/app/controllers/Joiner.scala

```diff
@@ -272,6 +282,7 @@ object Joiner extends Controller with ActivityTracking
   }
 
   def thankyou(tier: Tier, upgrade: Boolean = false) = SubscriptionAction.async { implicit request =>
+    implicit val resolution: TouchpointBackend.Resolution = TouchpointBackend.forRequest(PreSigninTestCookie, request.cookies)
```

Comment:
This is identical to [this line](https://github.com/guardian/membership-frontend/pull/1478/files#diff-071e948f4be1f1aa847ecc97e7a8110bR121). Could be pulled out into a function if it ends up being used on another page as well 🐔.